### PR TITLE
Rs 2022 06 26.1

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [sha:62ba0afe7179253e82b47c9989914ad42e56da54]{
 -- Page 10017, Offset 6725.
 -- 「……約束、したからな」
-"...I made a promise after all."
+"I made a promise, after all..."
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2.txt:4]: pause is really ugly in english. I say take it out.

moving ellipses to end is fine though

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt [hunk 2]
```diff
@@ -51,22 +52,23 @@ I'm giving myself a hard time.
...
 -- 朝の追いかけっこからここまで、心も体も疲れ切ってマトモに働こうとしていないようだ。
-After all the events between this morning's game of tag and now, my body and mind are exhausted to the point where they no longer function properly.
+After everything that's happened between this morning's game of tag and now,
+ my body and mind are so exhausted that they aren't even functioning properly.
 }
 [sha:9655a08893b76d3380588429b67c58cd37ceca69]{
 -- Page 425, Offset 6736.
 -- 「とにかく、まずは落ち着こう」
-"Anyway, I need to calm down first."
+"First things first, I need to calm down a bit."
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2.txt:61]: first: I

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt [hunk 3]
```diff
@@ -166,7 +169,8 @@ It's a little after eleven o'clock. This isn't the sort of truancy you can just
 [sha:53142bfa84200b7a851b775087ce15c74c2a333c]{
 -- Page 432, Offset 6758.
 -- 学校から家に連絡が入っていればアウトだが、こればかりはもうどうしようもない。
-If the school contacts my home it'll be game over for me, but there's no helping that now.
+If the school contacts the mansion it'll be game over for me,
+ but I can't do much about that now.
 }
 [sha:6efde8cc61f1f106d889fe54f6b3e5d911694195]{
 -- Page 432, Offset 6759.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2.txt:176]: 6759: 'telling-off' -> 'scolding'

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt [hunk 4]
```diff
@@ -206,7 +210,8 @@ I wish I could tell him more about the situation, but...
 [sha:20c8146e32d6fef8e5105a4af64e055cd68e9b67]{
 -- Page 436, Offset 6766.
 -- 「いやいや。それこそ救急車案件でしょ」
-"No, no. I couldn't do that unless it was a real emergency."
+// Assuming 救急車案件 is generic problem, unless it's actually the loony bin collection service?
+"No, scratch that. It'd only cause more problems than it solves."
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2.txt:214]: I'm not so sure about this new interpretation.

救急車案件 to me is lit. like "an ambulance matter" which sure sounds either: closer to the old version or "real cause for concern [for arihiko]". I can kinda see the latter interpretation in the current line, but I'd expect something more like "That'd only get him worried/asking questions/causing a stir"

Can you elaborate a little on your thoughts?

File: script/Arcueid/Day 4/04_00_ARC04_6_2DASH.txt [hunk 5]
```diff
@@ -294,12 +300,14 @@ You had let your guard down so much that I was seriously starting to worry that
...
 -- 「なんだよ、無防備なのはそっちだって同じだろ。
-"What? You were just as vulnerable as I was.
+"What are you on about, you were just as vulnerable as I was.
 }
 [sha:a63b9d08e1086d6b8cea11efaf15eea915bb4ec9]{
 -- Page 467, Offset 6856.
 -- 俺だって一度はおまえを殺してるんだぞ。今回だってそうしないとはかぎらないのに、お姫さまみたいにグースカ寝やがって。人間だからって甘く見るなよな」
-You know, I've already killed you once before. And yet there you were, sound asleep like a princess without any guarantee that I wouldn't do it again. Don't underestimate me just because I'm a human."
+I've already killed you once, remember?
+ You didn't have any guarantee I wouldn't do it again, but there you were, sound asleep like a princess.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2DASH.txt:309]: I won't do it again

**Review by Avuxo — APPROVED** [review]
otsukare

these files are so nostalgic

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 4/04_00_ARC04_6_2.txt:214]: Yeah, my read was along the lines of
> (それこそ)(救急車案件)(でしょ)
> (that in itself [telling Arihiko what's going down])(emergency)(would definitely be)

So, "telling him would be a problem in itself" -> context is trying to solve a problem (getting done for truancy), adjusted to idiom (cause more problems than it solves).

**Comment by rschlaikjer** [comment]
> these files are so nostalgic

second release and somehow I'm still scrambling to review the same damn portion of the game

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 4/04_00_ARC04_6_2.txt]: Seems like it is more literal, Arihiko would volunteer him for a vacation. Changed.

**Pull request merged**